### PR TITLE
Vietnamese translation

### PR DESCRIPTION
**Pull request by honghanhh**
As mentioned in https://github.com/huggingface/course/issues/292, I translated into Vietnamese.

**0 - Setup**
- [x] [`1.mdx`](https://github.com/huggingface/course/blob/main/chapters/vi/chapter0/1.mdx)

**1 - Transformer models**

- [x] [`1.mdx`](https://github.com/huggingface/course/blob/main/chapters/vi/chapter1/1.mdx)
- [x] [`2.mdx`](https://github.com/huggingface/course/blob/main/chapters/vi/chapter1/2.mdx)
- [x] [`3.mdx`](https://github.com/huggingface/course/blob/main/chapters/vi/chapter1/3.mdx)
- [x] [`4.mdx`](https://github.com/huggingface/course/blob/main/chapters/vi/chapter1/4.mdx)
- [x] [`5.mdx`](https://github.com/huggingface/course/blob/main/chapters/vi/chapter1/5.mdx)
- [x] [`6.mdx`](https://github.com/huggingface/course/blob/main/chapters/vi/chapter1/6.mdx)
- [x] [`7.mdx`](https://github.com/huggingface/course/blob/main/chapters/vi/chapter1/7.mdx)
- [x] [`8.mdx`](https://github.com/huggingface/course/blob/main/chapters/vi/chapter1/8.mdx)
- [x] [`9.mdx`](https://github.com/huggingface/course/blob/main/chapters/vi/chapter1/9.mdx)
- [x] [`10.mdx`](https://github.com/huggingface/course/blob/main/chapters/vi/chapter1/10.mdx)

**2 - Using 🤗 Transformers**
- [x] [`1.mdx`](https://github.com/huggingface/course/blob/main/chapters/en/chapter2/1.mdx)
- [x] [`2.mdx`](https://github.com/huggingface/course/blob/main/chapters/en/chapter2/2.mdx)
- [x] [`3.mdx`](https://github.com/huggingface/course/blob/main/chapters/en/chapter2/3.mdx)
- [x] [`4.mdx`](https://github.com/huggingface/course/blob/main/chapters/en/chapter2/4.mdx)
- [x] [`5.mdx`](https://github.com/huggingface/course/blob/main/chapters/en/chapter2/5.mdx)
- [x] [`6.mdx`](https://github.com/huggingface/course/blob/main/chapters/en/chapter2/6.mdx)
- [x] [`7.mdx`](https://github.com/huggingface/course/blob/main/chapters/en/chapter2/7.mdx)
- [x] [`8.mdx`](https://github.com/huggingface/course/blob/main/chapters/en/chapter2/8.mdx)

**3 - Fine-tuning a pretrained model**
- [x] [`1.mdx`](https://github.com/huggingface/course/blob/main/chapters/en/chapter3/1.mdx)
- [x] [`2.mdx`](https://github.com/huggingface/course/blob/main/chapters/en/chapter3/2.mdx)
- [x] [`3.mdx`](https://github.com/huggingface/course/blob/main/chapters/en/chapter3/3.mdx)
- [x] [`3_tf.mdx`](https://github.com/huggingface/course/blob/main/chapters/en/chapter3/3_tf.mdx)
- [x] [`4.mdx`](https://github.com/huggingface/course/blob/main/chapters/en/chapter3/4.mdx)
- [x] [`5.mdx`](https://github.com/huggingface/course/blob/main/chapters/en/chapter3/5.mdx)
- [x] [`6.mdx`](https://github.com/huggingface/course/blob/main/chapters/en/chapter3/6.mdx)

**4 - Sharing models and tokenizers**
- [x] [`1.mdx`](https://github.com/huggingface/course/blob/main/chapters/en/chapter4/1.mdx)
- [x] [`2.mdx`](https://github.com/huggingface/course/blob/main/chapters/en/chapter4/2.mdx)
- [x] [`3.mdx`](https://github.com/huggingface/course/blob/main/chapters/en/chapter4/3.mdx)
- [x] [`4.mdx`](https://github.com/huggingface/course/blob/main/chapters/en/chapter4/4.mdx)
- [x] [`5.mdx`](https://github.com/huggingface/course/blob/main/chapters/en/chapter4/5.mdx)
- [x] [`6.mdx`](https://github.com/huggingface/course/blob/main/chapters/en/chapter4/6.mdx)

**Events**
- [x] [`1.mdx`](https://github.com/huggingface/course/blob/main/chapters/en/event/1.mdx)
Please feel free to review it,  @lewtun!
Thanks :)

### CODE REVIEW

File: .github/workflows/quality.yml [hunk 1]
```diff
@@ -10,12 +10,12 @@ jobs:
...
\ No newline at end of file
+      - uses: actions/checkout@v2
+      - name: Set up Python 3.6
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.6
+      - name: Install Python dependencies
+        run: pip install black
+      - name: Run Quality check
+        run: make quality
```
Comment (tiena2cva) [inline .github/workflows/quality.yml:21]: don't format this file

Comment (honghanhh) [reply .github/workflows/quality.yml:21]: thanks for the suggestion!

File: .github/workflows/build_documentation.yml [hunk 2]
```diff
@@ -14,6 +14,6 @@ jobs:
       package: course
       path_to_docs: course/chapters/
       additional_args: --not_python_module
-      languages: ar bn de en es fa fr gj he hi it ja ko pt ru th tr zh-CN zh-TW
+      languages: ar bn de en es fa fr gj he hi it ja ko pt ru th tr zh-CN zh-TW vi
```
Comment (lewtun) [inline .github/workflows/build_documentation.yml:17]: Could we please order these alphabetically?

```suggestion
      languages: ar bn de en es fa fr gj he hi it ja ko pt ru th tr vi zh-CN zh-TW
```

File: .github/workflows/build_pr_documentation.yml [hunk 3]
```diff
@@ -16,5 +16,5 @@ jobs:
       package: course
       path_to_docs: course/chapters/
       additional_args: --not_python_module
-      languages: ar bn de en es fa fr gj he hi it ja ko pt ru th tr zh-CN zh-TW
-      hub_base_path: https://moon-ci-docs.huggingface.co
\ No newline at end of file
+      languages: ar bn de en es fa fr gj he hi it ja ko pt ru th tr zh-CN zh-TW vi
```
Comment (lewtun) [inline .github/workflows/build_pr_documentation.yml:19]:
```suggestion
      languages: ar bn de en es fa fr gj he hi it ja ko pt ru th tr vi zh-CN zh-TW
```

File: .github/workflows/quality.yml [hunk 4]
```diff
@@ -18,4 +18,4 @@ jobs:
     - name: Install Python dependencies
       run: pip install black
     - name: Run Quality check
-      run: make quality
\ No newline at end of file
+      run: make quality
```
Comment (lewtun) [inline .github/workflows/quality.yml:21]: This file shouldn't be changed, so I suggest resetting it with:

```
git checkout main -- .github/workflows/quality.yml
```

**Review by lewtun — APPROVED** [review]
Thanks a lot for this epic translation effort @honghanhh - a whole chapter in Vietnamese is amazing 🤯 !!

The format looks great to me - there's just two minor issues to be addressed before we merge

**Review by tiena2cva — APPROVED** [review]

**Comment by honghanhh** [comment]
Hi @lewtun, thanks for your feedback, I have updated up to chapter 3. Feel free to check it out. 
Thanks

**Comment by lewtun** [comment]
Thanks for iterating on this PR @honghanhh - it looks great! I'll deploy the changes today :)

**Review by lewtun — APPROVED** [review]

**Pull request merged**